cmake/GoogleCloudCppLibrary.cmake

```diff
@@ -307,9 +307,12 @@ function (google_cloud_cpp_add_ga_grpc_library library display_name)
                                      "include/google/cloud/${library}")
 
     google_cloud_cpp_add_pkgconfig(
-        ${library} "The ${display_name} C++ Client Library"
+        ${library}
+        "The ${display_name} C++ Client Library"
         "Provides C++ APIs to use the ${display_name}"
-        "google_cloud_cpp_grpc_utils" "${protos_target}")
+        "google_cloud_cpp_grpc_utils"
+        "${protos_target}"
+        ${shared_proto_dep_targets})
```

Comment:
nit: in these context these are pkg-config "modules" (the name of a `.pc` file).  We happen to use the same name for the module and the target and the library.

Reply:
Ack, thanks. That will help me talk about these things more intelligently.

I don't think I am going to change the variable names though :/